app/services/awards/find_service.rb

```diff
@@ -0,0 +1,28 @@
+module Awards
+  class FindService < ApplicationService
+    def initialize(options = { filter: nil, page: nil, order: nil })
+      @filter = options[:filter]
+      @page = options[:page]
+      @order = options[:order]
+    end
+
+    def call
+      if filter.present?
+        Award.includes(%i[categories country])
+             .where('name ILIKE ?', "%#{filter}%")
```

Comment:
https://stackoverflow.com/questions/25416667/rails-4-prevent-sql-injection-using-like-operator-in-sql-query
https://rorsecurity.info/portfolio/rails-sql-injection-like